bin/fuel-core-client/src/main.rs

```diff
@@ -15,6 +15,8 @@ enum TransactionCommands {
     /// Submit a JSON encoded transaction for inclusion in a block
     Submit { tx: String },
     /// Submit a JSON encoded transaction for a dry-run execution
+    Estimate { tx: String },
```

Comment:
```suggestion
    EstimatePredicates { tx: String },
```

we don't want to confuse people with thinking this is a general estimation API like what the SDK does.